docs/contributing/dependencies/index.md

```diff
@@ -55,12 +92,165 @@ version is **13 days**, and **13%** of repositories have adopted this version. R
 change. For more details read
 [Renovate documentation about Merge Confidence](https://docs.renovatebot.com/merge-confidence/).
 
-## Workflow
+## Management strategies
+
+:::success Our Goal: A sustainable review cadence
+
+The overarching purpose behind our use of these strategies is ensuring that our dependencies are as
+up-to-date as possible, while opening PRs for review at a frequency and level of complexity that is
+sustainable for all responsible teams.
 
 Renovate is currently scheduled to automatically create pull requests every 2 weeks. The goal of our
 dependency management process is for the teams to review and merge the opened pull requests in the
 same 2-week cadence. To avoid a large backlog of PRs and out-of-date packages accumulating.
```

Comment:
⛏️ Just noticed this but "2-week cadence. To" should be "2-week cadence -- to" (use a double-dash).

Reply:
Addressed with cf9320b75d9d56cf5291b3cf3b554bce77dda3e5.